app/view_objects/single_list_presenter.rb

```diff
@@ -0,0 +1,46 @@
+class SingleListPresenter
+
+  def initialize list
+    @list = list
+  end
+
+  def id
+    list.id
+  end
+
+  def title
+    list.title
+  end
+
+  def description
+    list.description
+  end
+
+  def created_at
+    date = list.created_at
+    date.strftime('%B %d, %Y')
+  end
```

Comment:
@vlasiak, I suggest to put formats into locales (create `formats` section in the root).